clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/RedisCacheConfig.groovy

```diff
@@ -43,6 +44,11 @@ import java.util.concurrent.TimeUnit
 @Configuration
 @ConditionalOnProperty('redis.connection')
 class RedisCacheConfig {
+  @Value('${redis.scheduler}')
```

Comment:
I think this will complain if the property doesn't exist :- change it to redis.scheduler:default and it will use default if the property is not set